lib/timing.mjs

```diff
@@ -0,0 +1,125 @@
+import process from 'node:process';
+import { table } from 'table';
+
+// Inspired by ESLint's timing.js
+// https://github.com/eslint/eslint/blob/main/lib/linter/timing.js
```

Comment:
[suggestion] I think a permalink with a commit ID is better than the `main` branch because `main` is changeable. E.g.

```suggestion
// https://github.com/eslint/eslint/blob/09bc2a88c00aa9a93c7de505795fc4e85b2e6357/lib/linter/timing.js
```